fastlane/spec/unused_options_spec.rb

```diff
@@ -3,6 +3,18 @@
     describe "No unused options" do
       let (:all_exceptions) { %w(pilot appstore cert deliver gym match pem produce scan sigh snapshot supply testflight mailgun testfairy ipa import_from_git hockey deploygate crashlytics artifactory appledoc slather screengrab download_dsyms) }
 
+      # https://github.com/bundler/bundler/issues/4368
```

Comment:
This spec is the only place that this problem occurs, so it felt good to apply it locally. More documentation would probably be good. I'll add some.